apps/examples/board_specific_demo/ameba_secure_world_demo/Kconfig

```diff
@@ -6,7 +6,7 @@
 config EXAMPLES_SECURE_WORLD
 	bool "Secure World example"
 	default n
-	depends on ARMV8M_TRUSTZONE && SE_AMEBA
+	depends on (ARMV8M_TRUSTZONE || AMEBASMART_TRUSTZONE) && SE_AMEBA
```

Comment:
Original one is chip architecture config (ARMV8M), but your code is chipset code.
I think it should be ARMV7A_TRUSTZONE

Reply:
Unlike ARMV8M, ARMV7A_TRUSTZONE config is not defined in Kconfig.
When ARMV7A_TRUSTZONE is added, Menuconfig does not show secure_world example config.